src/domains/ecommerce-common/02_security.tf

```diff
@@ -257,18 +257,6 @@ resource "azurerm_key_vault_secret" "notifications_service_api_key" {
   }
 }
 
-resource "azurerm_key_vault_secret" "payment_transactions_gateway_api_key" {
```

Comment:
this api key is used https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/aa0eabb4d0107fa92cd027cde1d6fa18e212ed6f/helm/values-prod.yaml#L191.
so this PR can be applied after the `transaction-service` no longer uses this api key, right?

Reply:
Yes, I thought I had already added the dependencies but I didn't, udpated them in the description right now :)